deployer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin:v1.2.0
+FROM openshift/origin:v1.5.0-alpha.2
```

Comment:
Agreed, we shouldn't need to really bump this unless we want to take advantage of some of the new oc/oadm features